packages/client/src/lib.js

```diff
@@ -97,9 +98,13 @@ class Web3Storage {
     onRootCidReady,
     onStoredChunk,
     maxRetries = MAX_PUT_RETRIES,
+    maxChunkSize = MAX_CHUNK_SIZE,
     wrapWithDirectory = true,
     name
   } = {}) {
+    if (maxChunkSize >= 104857600 || maxChunkSize < 1048576) {
```

Comment:
@akazwz now that we have constant, we should use them in this conditional and other below. Also, we should have a constant for the 100MB upper limit